appveyor.yml

```diff
@@ -60,13 +54,13 @@ install:
   - pip --version
 
   # upgrade setuptools and virtualenv
-  - pip install --upgrade setuptools virtualenv
+  - pip install --upgrade setuptools wheel virtualenv
 
   # install tox to run test suite in a virtual environment
   - pip install --upgrade tox
 
   # build wheel
-  - tox -e wheel
+  - pip wheel --no-deps --wheel-dir dist .
```

Comment:
I changed appveyor to build directly using pip instead of indirectly calling tox, because some environment variable was not getting through the tox sandboxing. I know I can use `passenv` but I didn't know exactly the name of the required env vars (doing `passenv=*` would have been even lazier). It's not really necessary to build the wheel inside a tox virtual environment. This works just fine.
The tests using the pre-built wheel are still being run inside tox.